src/iosrv/server_iv.c

```diff
@@ -865,8 +865,11 @@ iv_op_internal(struct ds_iv_ns *ns, struct ds_iv_key *key_iv,
 	int			rc;
 
 	rc = ABT_future_create(1, NULL, &future);
-	if (rc)
+	if (rc) {
+		if (sync->ivs_comp_cb)
```

Comment:
The 'sync' could be NULL here, right? For exmaple ds_iv_fetch() will pass in a NULL 'sync'.

Reply:
ah, right

Reply:
Good catch, fixed.